sass/components/_post.scss

```diff
@@ -2197,3 +2251,47 @@
         }
     }
 }
+
+@media (min-width: 768px) {
+    .post.post--compact.same--root.post--comment .post__content {
+        border-color: rgba(var(--center-channel-color-rgb), 0.2);
+    }
+
+    .post.a11y--active {
+        background: rgba(var(--center-channel-color-rgb), 0.08);
```

Comment:
Why do we need to change the background color depending on the width? Are the styles not coherent between mobile view and web view?

Reply:
good question .. will investigate

Reply:
I cannot find a good reason for that ... but since it was done this way before I would opt for leaving it that way for now